python/foxglove-sdk-examples/live-visualization/main.py

```diff
@@ -108,7 +108,7 @@ def main() -> None:
     tf_chan = FrameTransformsChannel("/tf")
     point_chan = PointCloudChannel("/pointcloud")
 
-    # Log dicts using JSON encoding
+    # Log dicts using JSON encoding. You can omit a schema if you specify message_encoding="json".
```

Comment:
@jtbandes ping'ed me on this so I'll share my thoughts:

I would remove this comment (about omitting a schema) from the example and encourage the practice of providing a schema. If you want to introduce a schema-less example then that would be a good place to go into more of the nuances of when you can do that. Keep this example focused on one thing.

As a non-python person I am not clear on the comment "Log dicts using JSON encoding" - this line does not mention json encoding anywhere. Is that the automatic choice when no message encoding is specified?